### PR TITLE
Added distributed CIFAR-10 example

### PR DESCRIPTION
**Pull request by mhbuehler**
This addresses issue #11

- Added a distributed training example for the CIFAR10 dataset
- Training is followed by validation after each epoch, and output metrics show ~70% test accuracy after 10 epochs
- Reorganized examples directory with mnist and cifar10 subdirectories and updated existing references to mnist files
- Updated documentation of examples to include instructions for running the CIFAR10 example and data pre-downloading for faster execution

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pytorch-operator/23)

### CODE REVIEW

**Comment by jose5918** [comment]
@mhbuehler Can you add a description about the changes that are being made?

**Comment by jose5918** [comment]
@mhbuehler Here is the error:
`error: the path "examples/multinode/configmap.yaml" does not exist`

**Comment by jose5918** [comment]
I really like the verbosity of this example. Great work.

File: examples/README.md [hunk 1]
```diff
@@ -1,11 +1,50 @@
...
-```
-kubectl apply -f multinode/
-```
+Here are examples of jobs we would aim to run with the operator.
 
-The configmap used in the example was created using the distributed training script found in this directory:
-```
-kubectl create configmap dist-train --from-file=dist_train.py
-```
+1. A simple example of distributed MNIST with pytorch on kubernetes:
```
Comment (jose5918) [inline examples/README.md]: @mhbuehler  The first example instructions I had were the plan for the operator. It doesn't actually use the operator. Can we remove instructions on this first example or have a readme for it in it's mnist directory? That or we can remove all the files from mnist aside from the configmap and pytorchjob
